### ISSUE TITLE
Time delay in rendering new view

### ISSUE THREAD

**Issue by janie-t** [WhareHauora/App#99]
Takes a long time to go to new route (see in URL bar)

**Comment by MeghanHughes** [comment]
Yeah this is because we are making all new api reqs every time the user changes to another page. @anna-marie-vos has been working on a fix where we make all the api calls in _app and then pass the data down into the pages components as props. (see https://github.com/WhareHauora/App/tree/av/_appjs - but also note that this branch does not include the work I am doing on finishing off the service workers and caching strategy so at the moment even if it does get completed we have to test it with that capacity too)
I think this would be a great thing to implement, but in general it is an optimisation that could be brought in in a future iteration? I don't think we expect users to spend much time navigating through the app on a regular basis? and our priority was to get the notifications working.
But we should ask for feedback from @Br3nda and @Kamikazilady if they agree with that priority trade-off or if we should spend a bit more time optimising the app speed now? 
I guess one bonus from making all those api calls is at least the user knows that the data is super up to date!

**Comment by MeghanHughes** [comment]
We are going to leave this as is.
Improvements to latency should be a future enhancement.
